packages/react-components/src/components/DevicePermissions/DomainPermissions.tsx

```diff
@@ -60,6 +64,13 @@ export interface CameraAndMicrophoneDomainPermissionsProps extends CommonDomainP
   strings?: CameraAndMicrophoneDomainPermissionsStrings;
 }
 
+/* @conditional-compile-remove(call-readiness) */
+const isSafari = (browser: string | undefined): boolean => {
+  /* @conditional-compile-remove(unsupported-browser) */
+  return browser === 'safari';
+  return /^((?!chrome|android|crios|fxios).)*safari/i.test(navigator.userAgent);
```

Comment:
I wouldn't do this at the pure component level and instead in `CallComposite/components/CallReadinessModal.tsx`

Reply:
updated